src/styles/CommentStyle.js

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+
+export const AddCommentBox = styled.div`
+  display:flex;
```

Comment:
How might you create this comment box with less styles but with the same result?

Reply:
" The Space betweeeeeeeennn" - dave matthews